### PR TITLE
JOSS Paper submission

### PR DESCRIPTION
**Pull request by ruthvikm**
- Final Draft version of the JOSS paper. 
- Created workflow to build the draft paper pdf
- [paper.pdf](https://github.com/user-attachments/files/17779387/paper.pdf)

### CODE REVIEW

File: joss-paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+# Statement of Need
+
+Scientific research often requires customized data-gathering instruments that can be integrated with a computing platform. 
+While some research devices can be connected to a PC, there are situations where more flexible, cost-effective solutions are necessary. 
+The Raspberry Pi, due to its affordability, size, and versatility, is a popular choice for such research. Applications for the Raspberry Pi 
+include rat licking behavior '@Longley:2017', greenhouse gas effect monitoring system '@Shah:2017',and many other applications '@Jolles:2021'. 
+However, interfacing with sensors and other electronic components in Java poses significant challenges. 
+
+Pi4Micronaut addresses this gap by providing a Java-based solution that integrates with Pi4J for GPIO (General Purpose Input/Output) control. 
+This allows researchers to use Java, a well-established programming language in scientific computing, to interact with hardware components on the Raspberry Pi,
```
Comment (kungfuchicken) [inline joss-paper/paper.md:49]: "`Java, a well-established programming language in scientific computing,`" citations supporting this claim would be good (https://doi.org/10.1002/cae.20217 etc.)

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
i think it's basically ready to ship.

**Review by kate-holdener — APPROVED** [review]
Looks good to me! Feel free to merge and submit to JOSS.

**Review by kungfuchicken — APPROVED** [review]
lgtm;

**Pull request merged**